google/mozc
# Fix wrong column order in aux dictionary

## Description

The rows for "対象外" & "対象内" in `aux_dictionary.tsv` have wrong column order.

## Issue IDs
Issue and/or discussion IDs related to this pull request.

## Steps to test new behaviors (if any)
A clear and concise description about how to verify new behaviors (if any).
 - OS: any
 - Steps:
   1. Input "対象外" or "対象内"
   2. Compose
   3. Confirm both of them are treated as single words

## Additional context
Add any other context about the pull request here.
